src/core/context/MainWrapper.tsx

```diff
@@ -61,24 +60,25 @@ const MainWrapper = ({ children }: { children: ReactNode }) => {
 
 export default MainWrapper;
 
-const ContainerOverlay = styled.div<{ isLight: boolean }>(({ isLight }) => ({
+const ContainerOverlay = styled('div')(({ theme }) => ({
   position: 'fixed',
   top: 0,
   width: '100%',
   zIndex: zIndexEnum.OVERLAY_MOBILE_TOOLTIP,
   height: 'calc(100vh - 282px)',
   background: 'rgba(52, 52, 66, 0.1)',
-  backdropFilter: isLight ? 'blur(2px)' : 'blur(4px)',
-  [lightTheme.breakpoints.between('mobile_375', 'table_834')]: {
+  backdropFilter: theme.palette.isLight ? 'blur(2px)' : 'blur(4px)',
+
+  [theme.breakpoints.down('table_834')]: {
```

Comment:
You should use `tablet_768` as `table_834` is a legacy breakpoint that shouldn't be used anymore. Same with `desktop_1194`

Reply:
Done